### PR TITLE
Documentation for Docker Registry 2.0, discussed in #926

### PR DESCRIPTION
**Pull request by IanSaunders**
https://github.com/mesosphere/marathon/issues/926#issuecomment-125509252

### CODE REVIEW

**Comment by kolloch** [comment]
I actually first wrote my answer to your email on the mailing list and then discovered this pull request. So I am pleasantly surprised to already see your pull request here :)

I'll have a look at this.

File: docs/docs/native-docker.md [hunk 1]
```diff
@@ -176,11 +176,84 @@ documentation for more details on how Docker handles networking.
...
+    $ cp docker.tar.gz /etc/
+    ```
+
+##### Step 2: Mesos/Marathon config
+
+1. Add the path to the gziped login credentials to your marathon app definition
+
+    ```bash
+    "uris": [
+       "file:///etc/docker.tar.gz"
```
Comment (kolloch) [inline docs/docs/native-docker.md]: Maybe add a comment here that you need to store the file somewhere which is accessible on all nodes where you want to start your apps. So either distribute the file to all these hosts or use S3 or some private repository.

**Comment by kolloch** [comment]
This is really great documentation. I am impressed. Thank you very much!

I added a comment you might want to address.

**Comment by IanSaunders** [comment]
@kolloch I have broken the changes into a separate file linked from the native docker docs. I also added a section on access to the file. Finally merged this into a single commit! Thanks for feedback!

**Comment by kolloch** [comment]
@IanSaunders Thanks a lot. Awesome docs :+1:

**Pull request merged**